docs/rabbitmq.config.example

```diff
@@ -246,6 +246,10 @@
    %%
    %% {memory_monitor_interval, 2500},
 
+   %% The total memory available can be calculated from the OS resources
+   %% - default option - or provided as a configuration parameter:
+   %% {total_memory_available_override_value, "5000MB"},
```

Comment:
We support `MiB`, `GiB` and such, so can change `rabbitmq.config.example` to use that.